.rubocop.yml

```diff
@@ -24,3 +24,4 @@ Style/StringLiteralsInInterpolation: { Enabled: true, EnforcedStyle: single_quot
 Style/TrailingCommaInArguments: { Enabled: true, EnforcedStyleForMultiline: consistent_comma } # enable
 Style/TrailingCommaInBlockArgs: { Enabled: true } # enable
 Style/WordArray: { Enabled: false } # disable
+Style/BlockDelimiters: { Enabled: true } # {...} for single-line blocks and do..end for multi-line blocks
```

Comment:
Just FYI, this rule also has many config options. The main one is `EnforcedStyle`. You can check how many violations we have in some of our services by pointing to this git commit in `package.json`. I suspect that we frequently use `braces_for_chaining`:

<img width="830" alt="image" src="https://github.com/angellist/eslint-config-angellist/assets/577441/30f3d41c-1123-4bd8-ba11-94dd89136bed">

https://docs.rubocop.org/rubocop/cops_style.html